app/views/auctions/_list_item.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="usa-grid">
     <div class='usa-width-one-whole issue-label'>
       <h3 class='issue-title'>
-        <span class="usa-label-big <%= auction_label_class(auction) %>"><%= auction_label(auction) %></span> <a href="<%= auction_path(auction) %>"><%= auction.title %></a>
+        <span class="usa-label-big <%= auction.label_class %>"><%= auction.label %></span> <a href="<%= auction_path(auction) %>"><%= auction.title %></a>
```

Comment:
Another dumb question, but would you then wrap the auction in an `AuctionShow` class here to use that code in these partials? I'm just trying to make sure I correctly understand how we don't duplicate methods between `AuctionIndex` and `AuctionShow` when the same information is displayed on both pages.